### ISSUE TITLE
Understanding Blind Screen-Reader Users’ Experiences of Digital Artboards

### ISSUE THREAD

**Issue by Wotipati** [Wotipati/Blind_Accessibility_Papers#401]
### Links
- Paper: https://dl.acm.org/doi/10.1145/3411764.3445242

### Abstract
Two-dimensional canvases are the core components of many digital productivity and creativity tools, with “artboards” containing objects rather than pixels. Unfortunately, the contents of artboards remain largely inaccessible to blind users relying on screen-readers, but the precise problems are not well understood. This study sought to understand how blind screen-reader users interact with artboards. Specifically, we conducted contextual interviews, observations, and task-based usability studies with 15 blind participants to understand their experiences of artboards found in Microsoft PowerPoint, Apple Keynote, and Google Slides. Participants expressed that the inaccessibility of these artboards contributes to significant educational and professional barriers. We found that the key problems faced were: (1) high cognitive loads from a lack of feedback about artboard contents and object state; (2) difficulty determining relationships among artboard objects; and (3) constant uncertainty about whether object manipulations were successful. We offer design remedies that improve feedback for object state, relationships, and manipulations.

<img width="411" alt="ScreenShot 2021-07-03 1 08 36" src="https://user-images.githubusercontent.com/26996041/124301791-43de7b80-db9b-11eb-9570-f33de164232b.png">

- スクリーンリーダーを使用する視覚障害者にとって，スライド作りなど，アートボード上に描画するツールは使いづらい
- インタビューとユーザ実験を通して，現在のツールの問題点を指摘
